Sources/Testing/ExitTests/ExitTest.Condition.swift

```diff
@@ -0,0 +1,148 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024–2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+private import _TestingInternals
+
+@_spi(Experimental)
+#if SWT_NO_EXIT_TESTS
+@available(*, unavailable, message: "Exit tests are not available on this platform.")
+#endif
+extension ExitTest {
+  /// The possible conditions under which an exit test will complete.
+  ///
+  /// Values of this type are used to describe the conditions under which an
+  /// exit test is expected to pass or fail by passing them to
+  /// ``expect(exitsWith:observing:_:sourceLocation:performing:)`` or
+  /// ``require(exitsWith:observing:_:sourceLocation:performing:)``.
+  public struct Condition: Sendable {
+    /// An enumeration describing the possible requirements for an exit test.
+    private enum _Kind: Sendable, Equatable {
+      /// The exit test must exit with a particular exit status.
+      case statusAtExit(StatusAtExit)
```

Comment:
Even if you do go with the current type name (`StatusAtExit`), I'm not so sure you need to repeat the "AtExit" part in the name of this `case`
```suggestion
      case status(StatusAtExit)
```

Reply:
It felt weird typing just `status` elsewhere. In any case, this is a private enum, so we don't need to worry too much about it.